config/app_config.go

```diff
@@ -0,0 +1,32 @@
+package config
```

Comment:
Couldn't this code be moved directly in `hub/options.go`?

Reply:
This would imply that config is tied to only the *hub* package? my thoughts behind this was an independent package who handles configuration and could be used anywhere ..say we need to somehow add another package in the future.. don't you think dependency on the *hub* package is somewhat bringing unnecessary baggage ?

Reply:
But as the hub package depends of the config package that depends of viper etc, it doesn't fix this issue right now, isn't it?